composer.json

```diff
@@ -42,6 +44,9 @@
         "florianv/swap": "Exchange rates library for PHP",
         "psr/cache-implementation": "Used for Currency caching"
     },
+    "conflict": {
+        "florianv/swap": "<3.0"
```

Comment:
Is it really needed? It's impossible to install this package with `florianv/swap<3.0`.